MapleServer2/Tools/HomeManager.cs

```diff
@@ -20,7 +20,12 @@ public HomeManager()
             SaveLoop();
         }
 
-        public void AddHome(Home home) => HomeList[home.Id] = home;
+        public void AddHome(Home home)
+        {
+            home.InstanceId = InstanceCounter;
+            HomeList[home.Id] = home;
+            IncrementCounter();
```

Comment:
Can we just make `IncrementCounter()` return the current count so we can just set it directly all in one motion? Like `home.InstanceId = IncrementCounter();`